packages/core/src/renderer/getStyleSheetForBucket.ts

```diff
@@ -123,6 +153,12 @@ function findInsertionPoint(
   while (index >= 0) {
     const styleElement = styleElements.item(index);
 
+    if (isSameInsertionKey(styleElement, targetBucket, metadata)) {
```

Comment:
There's no need for branching logic here, the priority comparer is just another aspec of the comparing?

```ts
  const priorityComparer = (el: HTMLStyleElement) => priority - Number(el.getAttribute('data-priority'));
  let comparer: (el: HTMLStyleElement) => number = el => {
    if (isSameInsertionKey(el, targetBucket, metadata)) {
      return priorityComparer(el);
    }

    return targetOrder - styleBucketOrderingMap[el.getAttribute(DATA_BUCKET_ATTR) as StyleBucketName];
  }
```

Reply:
yeah, you're correct 👍